gusto/forcing.py

```diff
@@ -160,57 +188,25 @@ class IncompressibleForcing(Forcing):
     Forcing class for incompressible Euler Boussinesq equations.
     """
 
-    def _build_forcing_solver(self):
-        """
-        Only put forcing terms into the u equation.
-        """
-
-        state = self.state
-        self.scaling = Constant(1.)
-        Vu = state.spaces("HDiv")
-        W = state.W
-
-        self.x0 = Function(W)   # copy x to here
+    def pressure_gradient_term(self):
+        _, p0, _ = split(self.x0)
+        L = div(self.test)*p0*dx
+        return L
 
-        u0,p0,b0 = split(self.x0)
+    def gravity_term(self):
+        _, _, b0 = split(self.x0)
+        L = b0*inner(self.test,self.state.k)*dx
```

Comment:
Set up a branch that turns off the E231 ignore.  I'll wait until this PR lands to save you a PITA merge.